apps/web/src/app/(signing)/sign/[token]/page.tsx

```diff
@@ -44,6 +44,10 @@ export default async function SigningPage({ params: { token } }: SigningPageProp
     return notFound();
   }
 
+  if (document?.status === 'COMPLETED') {
```

Comment:
You should be able to do `document.status` rather than `document?.status` since we perform our undefined check above 😄

Reply:
I fixed it